dkc/core/rest/folder.py

```diff
@@ -39,22 +65,70 @@ class FoldersFilterSet(filters.FilterSet):
     parent = IntegerOrNullFilter(required=True)
 
 
+class FolderPermissionGrantSerializer(serializers.Serializer):
+    name = serializers.CharField(required=True)
```

Comment:
Non-actionable comment for posterity: if we also wanted to support an ID-based API, this is forward compatible. We would just make `name` not required, and add an `id` field, and one of them would have to be set.